docs-ref-conceptual/manage-cloud-sets-azure-cli.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Managing Cloud Sets with Azure CLI 2.0
+description: Create, log in, and manage different cloud sets with the Azure CLI 2.0.
+keywords: Azure CLI 2.0, Azure, cloud sets, datacenters
+cloud: ???

```

Comment:
I've never seen a "cloud" tag, it's not in any of my articles (or dotnet, or java), and it's not in the contributor's guide. It might be outdated. I would recommend removing it.